GVFS/GVFS.Installer/CreateMacInstaller.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+CONFIGURATION=$1
+if [ -z $CONFIGURATION ]; then
+  CONFIGURATION=Debug
+fi
+
+PACKAGEVERSION=$2
+if [ -z $PACKAGEVERSION ]; then
+  PACKAGEVERSION="0.0.0.1"
+fi
+
+BUILDOUTPUTDIR=$3
+if [ -z $BUILDOUTPUTDIR ]; then
+  echo "Build output directory not specified"
+  exit 1
+fi
+
+STAGINGDIR=$BUILDOUTPUTDIR"Staging"
+VFSFORGITDESTINATION="usr/local/vfsforgit"
+KEXTDESTINATION="/Library/Extensions"
+INSTALLERPACKAGENAME="VFSForGit.$PACKAGEVERSION"
+INSTALLERPACKAGEID="com.vfsforgit.pkg"
```

Comment:
@ethomson can you remind me which domain(s) we registered with the new name?

Reply:
It is indeed vfsforgit.com.  Thanks for the reminder, I should have some time this sprint to get the site moved over.